.github/workflows/Manager-CI.yaml

```diff
@@ -161,3 +161,9 @@ jobs:
         if: always()
         run: |
           kubectl logs deployment/event-bridge -n event-bridge-manager
+      - name: Upload test logs
+        if: always()
```

Comment:
Is "if always" necessary here as it will run every time anyway?

Reply:
It is needed because the test failures in Maven build fails the GH action script. So we want to store the logs in case of test failure.